MarijaSaysHello/Task3/TaskRunner.cs

```diff
@@ -0,0 +1,231 @@
+﻿using System.Globalization;
+
+namespace Marija_SaysHello.Task3
+{
+    public class TaskRunner
+    {
+        public static void Task3Runner()
+        {
+            Number1();
+            Number2();
+            Number3();
+            Number4();
+            Number5();
+            Number6();
+            Number7();
+            Number8();
+            Number9();
+        }
+
+        public static void Number1()
+        {
+            var num = 20;
+            num += 5;
+
+            Console.WriteLine($"Variable: var {num}");
+        }
+
+        public static void Number2()
+        {
+            //uznat na schiot mesiacev i dnej v godu
+            //uznat nuzhno li sdelat proverku esli vvediosh ne chetiriohznachnoe chislo
+            Console.Write("Hi! Please enter the four-digit number:  ");
+
+            string userInput = Console.ReadLine();
+            int n;
+
+            bool isParsable = Int32.TryParse(userInput, out n);
+            if (!isParsable)
+            {
+                Console.WriteLine("Could not be parsed.");
+                return;
+            }
+
+            int daysInYear = 365;
+
+            int years = n / daysInYear;
+
+            int daysInMonth = 30;
+            int months = (n - years * daysInYear) / daysInMonth;
+
```

Comment:
Please remove empty lines. Usually we add empty lines after blocks of code with brackets ({}) or to split some steps/logical blocks